orchestrator/android/README_kotlin.md

```diff
@@ -27,16 +27,15 @@ Add the orchestrator dependency to your module (often called app). In your **mod
 ```gradle
 dependencies {
     ...
-    def dc_version = '1.1.0'
+    def dc_version = "1.1.0"
```

Comment:
this might go up :/

Reply:
it automatically goes up through the script when we update the SDK
./update_orch.sh :)